disco_aws_automation/disco_vpc_gateways.py

```diff
@@ -0,0 +1,291 @@
+"""
```

Comment:
@kelibezhani can you look over this file, I skimmed it looks good to me, but I think you are more familiar with this.